extensions/ql-vscode/src/common/vscode/vscode-app.ts

```diff
@@ -6,14 +6,19 @@ import { AppEventEmitter } from "../events";
 import { extLogger, Logger } from "../logging";
 import { Memento } from "../memento";
 import { VSCodeAppEventEmitter } from "./events";
+import { ExtensionCommandManager } from "../commands";
+import { initializeVSCodeCommandManager } from "./commands";
 
 export class ExtensionApp implements App {
```

Comment:
Unrelated to this PR, but I wonder why we call this `ExtensionApp` when the file is called `vscode-app`.